interfaces/apparmor/spec_test.go

```diff
@@ -506,6 +506,33 @@ func (s *specSuite) TestApparmorOvernameSnippets(c *C) {
 	c.Assert(updateNS[0], Equals, profile)
 }
 
+func (s *specSuite) TestApparmorExtraLayouts(c *C) {
+	snapInfo := snaptest.MockInfo(c, snapTrivial, &snap.SideInfo{Revision: snap.R(42)})
+	snapInfo.InstanceKey = "instance"
+
+	restore := apparmor.SetSpecScope(s.spec, []string{"snap.some-snap_instace.app"})
+	defer restore()
+
+	extraLayouts := []snap.Layout{
+		{
+			Path: "/test",
+			Bind: "/usr/home/test",
+			Mode: 0755,
+		},
+	}
+
+	s.spec.AddExtraLayouts(snapInfo, extraLayouts)
+	c.Assert(s.spec.Snippets(), HasLen, 0)
```

Comment:
(nitpick) maybe a small comment here why the snippets are 0 here? (but maybe it just shows my ignorance)

Reply:
Removed this, it didn't make sense in that scenario :-) Thanks for spotting it!